### ISSUE TITLE
Enhancement: Ability to select a folder for recordings.

### ISSUE THREAD

**Issue by SalieriC** [BlitzKraig/fvtt-VoiceActor#5]
Personally, I do not like having much stuff in Foundrys data folder, I like it much better to have it sorted in my worlds folders. A way to select a folder (scope world) where the sound files are placed would greatly enhance the user experience for me.

**Comment by BlitzKraig** [comment]
Agreed, I will add a module setting to select a recording directory.
